setup.py

```diff
@@ -101,7 +101,7 @@
       packages=find_packages(exclude=["*.tests", "*.tests.*",
                                       "tests.*", "tests"]),
       install_requires=[
-          'torch==0.4.0',
+          'torch==0.4.1',
```

Comment:
>=0.4.0?

Reply:
what's the rationale for allowing 0.4.0 still?

Reply:
That's also a valid point, if the version matters for our support. I could go either way here.

Reply:
For minor versions of the key library that we depend on, this seems a bit inflexible. But, if you feel strongly, go ahead.

Reply:
The only other thing I'd add is that if we have a lower bound, we should have an upper bound too.  Not sure if that should be `0.4.2`, `0.5.0`, or `1.0.0`.